Neo4jClient/GraphClient.cs

```diff
@@ -23,7 +23,7 @@
 
 namespace Neo4jClient
 {
-    public class GraphClient : IRawGraphClient, IInternalTransactionalGraphClient, IDisposable

```

Comment:
Ideally - make changes that only apply to the code you're doing - I'm pretty certain this is R#, as it points out to me that it's not needed as well :)